<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="image_topic" default="/camera/rgb/image_raw" />
    <arg name="storage_folder" default="$(find turtlebot_transfer_learning)/data" />
  
    <node
      name="camera_controller"
      pkg="image_view"
      type="image_saver"
      output="screen">
        <remap from="image" to="$(arg image_topic)"/>
        <param name="save_all_image" value="false" />
        <param name="filename_format" value="$(arg storage_folder)/image_%04d.png" />
    </node>
</launch>